config/locales/simple_form.en.yml

```diff
@@ -4,7 +4,7 @@ en:
     "no": "No"
     required:
       text: "required"
-      mark: '*'
+      mark: ""
```

Comment:
Same question: do you prefer not to display the user when a field is required? Usually that is considered good usability practice

Reply:
Done in sha: a2fa585